graf2d/asimage/src/TASImage.cxx

```diff
@@ -1118,7 +1119,7 @@ void TASImage::FromPad(TVirtualPad *pad, Int_t x, Int_t y, UInt_t w, UInt_t h)
             memcpy(fImage->alt.argb32, itmp->fImage->alt.argb32, sz*4);
          }
       }
-      delete gVirtualPS;
+      tclass_TImageDump->GetDelete()(gVirtualPS);
```

Comment:
This change is not needed as `TImageDump` properly derived from `TVirtualPS`.
```suggestion
      delete gVirtualPS;
```

Reply:
fair enough. Actually the delete in the dictionary is the regular one also.